src/widgets/widgets.ts

```diff
@@ -11,6 +11,7 @@ import React from 'react';
 
 import { AreaType } from '../types/area';
 import { ShapeType } from '../types/shape';
+import { boolean } from 'zod';
```

Comment:
Remove this import, it should not be used!

Reply:
Yeah that came in automatically when I was trying out the different types of boolean, thanks for catching it